tests/v2_validation/cattlevalidationtest/core/common_fixtures.py

```diff
@@ -2066,9 +2066,19 @@ def check_round_robin_access_lb_ip(container_names, lb_ip, port,
 
     for n in range(0, len(con_hostname)):
         if headers is not None:
-            r = requests.get(url, headers=headers)
+            try:
+                r = requests.get(url, headers=headers)
+            except requests.ConnectionError:
+                logger.info("Connection Error - " + url)
+                time.sleep(5)
```

Comment:
this sleep was intended to be used to wait until the lb is ready, because we saw some connection error exceptions when testing services/ingresses.

@sangeethah Suggested that i should use `wait_until_lb_ip_is_active` function to make sure the LB is up and running, i will modify the PR with these changes.